README.md

```diff
@@ -4,16 +4,55 @@ A service for issuing [RFC 3161 timestamps](https://datatracker.ietf.org/doc/htm
 
 Timestamps conform to the [RFC 3628 policy](https://datatracker.ietf.org/doc/html/rfc3628).
 
-### Prerequisites
+## Security model
 
-On MacOS, we currently require the installation of `openssl`. 
+[Trusted timestamping](https://en.wikipedia.org/wiki/Trusted_timestamping) is a process that has been around for some time.
+It provides a timestamp record of when a documented was created or modified.
+
+A timestamp authority creates signed timestamps using public key infrastructure. The operator of the timestamp
+authority must secure the signing key material to prevent unauthorized timestamp signing.
+
+A timestamp authority should also verify its own clock. We provide a configuration to periodically check the current
+time against well-known NTP sources.
+
+## Timestamping within Sigstore
+
+Timestamps are a critical component of [Rekor](https://github.com/sigstore/rekor), Sigstore's signature transparency log.
+Timestamps are used to verify short-lived certificates. Currently, the timestamp comes from Rekor's own internal clock,
+which is not externally verifiable. Using signed timestamps issued from timestamp authorities mitigates the risk of
+Rekor's clock being manipulated.
+
+As a artifact signer, you can:
+
+* Generate a signature over an artifact
+* Fetch a timestamp for that signature (more below in "What to sign")
+* Upload the signature, artifact hash, and certificate to Rekor (hashedrekord record type)
+* Upload the timestamp to Rekor (rfc3161 record type)
+   * This step is important because it makes the timestamps publicly auditable
```

Comment:
In future work, will timestamp-authority handle this itself?

Reply:
Someone asked about this awhile ago. My thought was to make this always client-driven, but maybe we make it configurable? What do you think?